File: Week_0.1/q11.c
#include <stdio.h>

int main() {
    int i,n, ctr = 0;
    int arr[100];

    printf("Enter the size of array : ");
    scanf("%d", &n);

    printf("Enter the values of array: ");
    for(i = 0; i < n; i++) {
        scanf("%d", &arr[i]);
    }

    for(i = 0; i < n; i++) {
        if(arr[i] % 2 == 0) {
            ctr++;
        }
    }

    printf("Number of even numbers in the given array is %d\n", ctr);
}

File: Week_3/q11.js
var readlineSync = require('readline-sync')

let limit = readlineSync.question("Enter the size of arrays ")
let a = []

console.log("Enter the values of Array");
for(let i = 0; i < limit; i++) {
    a[i] = Number(readlineSync.question())
}

let even = a.filter(x => x%2 == 0)
console.log(`Number of even numbers in the given array is ${even.length}`);